### PR TITLE
apply phase one name change

### PR DESCRIPTION
**Pull request by waarg**
[AB#8752](https://dev.azure.com/jitsuin/0629f48c-3979-4bbc-9026-cb06b3dfd0ae/_workitems/edit/8752)

<sub> Mention [stepsize] in a comment if you'd like to report some technical debt. See examples [here](https://app.stepsize.com/api/demo-pr-redirect).</sub>

### CODE REVIEW

File: api/go.mod [hunk 1]
```diff
@@ -9,7 +9,7 @@ require (
 	github.com/envoyproxy/protoc-gen-validate v0.10.1
 	github.com/google/uuid v1.3.1
 	github.com/grpc-ecosystem/grpc-gateway/v2 v2.17.1
-	github.com/rkvst/go-rkvstcommon v0.4.0
+	github.com/datatrails/go-datarails-common v0.4.0
```
Comment (eccles) [inline api/go.mod:12]: 0.9.0

File: api/go.sum [hunk 2]
```diff
@@ -54,8 +54,8 @@ github.com/pmezard/go-difflib v1.0.0 h1:4DBwDE0NGyQoBHbLQYPwSUPoCMWR5BEzIk/f1lZb
 github.com/pmezard/go-difflib v1.0.0/go.mod h1:iKH77koFhYxTK1pcRnkKkqfTogsbg7gZNVY4sRDYZ/4=
 github.com/prashantv/gostub v1.1.0 h1:BTyx3RfQjRHnUWaGF9oQos79AlQ5k8WNktv7VGvVH4g=
 github.com/prashantv/gostub v1.1.0/go.mod h1:A5zLQHz7ieHGG7is6LLXLz7I8+3LZzsrV0P1IAHhP5U=
-github.com/rkvst/go-rkvstcommon v0.4.0 h1:kt0F7eP73ybfSUkVbWXJtFm3XbIO0FKvuSjBjs0Q0+E=
-github.com/rkvst/go-rkvstcommon v0.4.0/go.mod h1:M0Mi6v/2Jgbc5se2zWKbShZC9bTVI7eFQE/xTi5Q9J8=
+github.com/datatrails/go-datatrails-common v0.4.0 h1:kt0F7eP73ybfSUkVbWXJtFm3XbIO0FKvuSjBjs0Q0+E=
+github.com/datatrails/go-datatrails- v0.4.0/go.mod h1:M0Mi6v/2Jgbc5se2zWKbShZC9bTVI7eFQE/xTi5Q9J8=
```
Comment (eccles) [inline api/go.sum:58]: was this changed manually ? Normally one changes go.mod and then executes 'go mod tidy', 'go mod verify' but I presume you dont have go installed locally

**Review by eccles — CHANGES_REQUESTED** [review]

File: api/go.mod (hunk 1 above)
Comment (waarg) [reply api/go.mod]: updated thanks

File: api/go.sum (hunk 2 above)
Comment (waarg) [reply api/go.sum]: manually updated but had issues running go mod as discussed

File: taskfiles/Taskfile_apis.yml [hunk 3]
```diff
@@ -160,11 +160,11 @@ tasks:
       #
       PROTO_GEN_GO_DIR:
         # NOTICE: You need to set GOCACHE for any go commands
-        sh: echo $(GOCACHE=/tmp/jitsuin/rkvstcommon-api/api go list -f {{"{{"}}.Dir{{"}}"}} google.golang.org/protobuf/cmd/protoc-gen-go )
+        sh: echo $(GOCACHE=/tmp/jitsuin/datatrails-common-api/api go list -f {{"{{"}}.Dir{{"}}"}} google.golang.org/protobuf/cmd/protoc-gen-go )
```
Comment (eccles) [inline taskfiles/Taskfile_apis.yml:163]: jitsuin?

**Review by eccles — CHANGES_REQUESTED** [review]

**Review by eccles — APPROVED** [review]

**Pull request merged**